Packs/XSOAR_Engineer_Training/Integrations/XSOAREngineerTraining/README.md

```diff
@@ -0,0 +1,156 @@
+
+#### Base Command
+
+`ad-get-user`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| dn | The Distinguished Name of the user in which to return information. | Optional | 
+| username | Queries users by the samAccountName attribute. | Optional | 
+| email | Queries by the user's email address. | Optional | 
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| ActiveDirectory.Users.dn | unknown | The distinguished name of the user. | 
+| ActiveDirectory.Users.displayName | unknown | The display name of the user. | 
+| ActiveDirectory.Users.name | unknown | The common name of the user. | 
+| ActiveDirectory.Users.sAMAccountName | unknown | The sAMAccountName of the user. | 
+| ActiveDirectory.Users.userAccountControl | unknown | The account control flag of the user. | 
+| ActiveDirectory.Users.mail | unknown | The email address of the user. | 
+| ActiveDirectory.Users.manager | unknown | The manager of the user. | 
+| ActiveDirectory.Users.memberOf | unknown | Groups for which the user is a member. | 
+| Account.DisplayName | unknown | The display name of the user. | 
+| Account.Groups | unknown | Groups for which the user is a member. | 
+| Account.Manager | unknown | The manager of the user. | 
+| Account.ID | unknown | The distinguished name of the user. | 
+| Account.Username | unknown | The samAccountName of the user. | 
+| Account.Email | unknown | The email address of the user. | 
+
+### ad-expire-password
+
+***
+Expires the password of an Active Directory user.
+
+#### Base Command
+
+`ad-expire-password`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| username | The username (samAccountName) of the user to modify. | Required | 
+
+#### Context Output
+
+There is no context output for this command.
+
+### ad-set-new-password
+
+***
+Sets a new password for an Active Directory user.
+
+#### Base Command
+
+`ad-set-new-password`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| username | The username of the account to disable (sAMAccountName). | Required | 
+| password | The password to set for the user. | Required | 
+
+#### Context Output
+
+There is no context output for this command.
+
+### siem-search
+
+***
+Searches the simulated SIEM for events.
+
+#### Base Command
+
+`siem-search`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| query | The query to execute against the SIEM. | Required | 
+| result_type | Type of result to return for this siem integration. Possible values are: email, hosts. | Optional | 
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| SIEM.Result | unknown | The results of the SIEM search. The results are a JSON array, in which each item is a SIEM event. | 
```

Comment:
if the result is an array, the type should be an array.
Please add it to the .yml as well.
```suggestion
| SIEM.Result | Array | The results of the SIEM search. The results are a JSON array, in which each item is a SIEM event. | 
```

Reply:
Done